### DISCUSSION TITLE
Adding support for getting single field values in @mantine/form

### DISCUSSION THREAD

**Discussion by anthonyalayo** [mantinedev/mantine#7201] [Feature requests]
Going through the docs and code with use-form:
https://mantine.dev/form/use-form/#values

I can see that there is the ability to get all form values, but not a single form value:
```
// get current form values
form.getValues();

// Set all form values
form.setValues(values);

// Set all form values using the previous state
form.setValues((prev) => ({ ...prev, ...values }));

// Set value of single field
form.setFieldValue('path', value);

// Set value of nested field
form.setFieldValue('user.firstName', 'Jane');
```

This would be a great API to have for cases where you need to run logic on particular form fields. In my use case, I am trying to set a hidden input from a particular form field (since the javascript and html formatting is different due to an input mask).

**Comment by gary-menzel** [comment]
If you can set a value with a path you need to be able get a value that way as well.  You already have a path traversal function to be able to set it - you should just provide similar for getting a value.

In my case, my forms are heavily dynamic and driven from configuration.  I have to write an ancillary function to get with a path when it should just be part of the form API.

**Comment by VaZark** [comment]
Seconding this. 

I have a nested component that triggers actions based on the selected value. Sometimes this value is from a nested path of a form. It would be simpler if we could traverse and get the value based on the exact form nested path
